pkg/kernelcache/kext.go

```diff
@@ -2,69 +2,220 @@ package kernelcache
 
 import (
 	"bytes"
+	"encoding/binary"
 	"fmt"
+	"io"
 	"os"
-
-	"github.com/blacktop/go-macho"
-	"github.com/blacktop/go-plist"
 )
 
-type prelinkInfo struct {
-	PrelinkInfoDictionary []CFBundle `plist:"_PrelinkInfoDictionary,omitempty"`
+// KextList lists all the kernel extensions in the kernelcache
+func KextList(kernel string) error {
+	kc, err := NewKernelCache(kernel)
+	if err != nil {
+		return err
+	}
+	defer kc.Close()
+
+	prelink, err := kc.PrelinkInfoDict()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("FOUND:", len(prelink.PrelinkInfoDictionary))
+	prelink.ForeachBundle(func(bundle * CFBundle) error {
```

Comment:
Check failure on line 25 in pkg/kernelcache/kext.go

 
GitHub Actions
/ build (windows-latest)

pkg/kernelcache/kext.go#L25
undefined: CFBundle